packages/presentation/pubspec.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
   flutter_svg: ^1.1.4
   shimmer: ^2.0.0
   flutter_rating_bar: ^4.0.1
+  readmore: ^2.1.0
```

Comment:
why do u need it?

Reply:
For expand and hide text in synopsis

Reply:
can we expand/collapse without this library?

Reply:
We can, but using this library is faster and completes all my needs.
Why shouldn't I use it?